README.md

```diff
@@ -6,21 +6,17 @@ Discord Rich Presence integration for the C* Music Player (`cmus`).
 
 ## Installing
 
-- If it isn't already on your system, install `rust`, `cargo`, and `git`. On \*nix based systems this should be as simple as installing it from your package manager.
+- If it isn't already on your system, install `rust`, and `cargo`. On \*nix based systems this should be as simple as installing it from your package manager.
```

Comment:
Should still require git.

Reply:
The users (people who have already customized their system, know about *nix enough to use cmus, but not cargo/rust) should have this installed already.

Reply:
You never know :man_shrugging:, you could also add info about building from zip/tar